src/components/tl/ucp/reduce_scatterv/reduce_scatterv.h

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2021-2022.  ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+#ifndef REDUCE_SCATTERV_H_
+#define REDUCE_SCATTERV_H_
+#include "../tl_ucp_reduce.h"
```

Comment:
isn't it src/componenets/tl/ucp in include path? do we need relative path for this header?

Reply:
we don't need relative indeed since we build the whole TL from tl/ucp folder